py/mpconfig.h

```diff
@@ -329,6 +329,9 @@
 // Convenience definition for whether any inline assembler emitter is enabled
 #define MICROPY_EMIT_INLINE_ASM (MICROPY_EMIT_INLINE_THUMB || MICROPY_EMIT_INLINE_XTENSA)
 
+// Convenience definition for whether any native or inline assembler emitter is enabled
+#define MICROPY_EMIT_ANY_ASM (MICROPY_EMIT_NATIVE || MICROPY_EMIT_INLINE_ASM)
```

Comment:
Yes, it's probably a good idea to make this define.  But, I'm just thinking if there's a better name for it, because "any asm" could be mistaken for meaning "any inline asm" (which is already what `MICROPY_EMIT_INLINE_ASM` is for).

Maybe `MICROPY_EMIT_MACHINE_CODE`?